toolboxes/feel/feelmodels/levelset/levelset.hpp

```diff
@@ -409,6 +410,10 @@ class LevelSet : public LevelSetBase<ConvexType, BasisType, PeriodicityType, Bas
         return hana::concat( super_type::allMeasuresQuantities( prefix ), hana::make_tuple(
                     ModelMeasuresQuantity( prefix, "velocity-com", std::bind( eigenToVec, std::bind( &self_type::velocityCOM, this ) ) )
                     ) );
+#endif
+        return Feel::FeelModels::modelMeasuresQuantities( super_type::allMeasuresQuantities( prefix ),
+                                                          modelMeasuresQuantity(prefix, "velocity-com", std::bind( &self_type::velocityCOM, this ) )
```

Comment:
could it be more explicit than "velocity-com" ?

Reply:
com == center of mass ?

Reply:
I don't know, it's level set code of @metivett

Reply:
it is probably center of mass.
if we choose this kind of naming, maybe we could use it everywhere and document it

Reply:
Yes it is refers to the "center-of-mass" velocity (or position). Would you prefer another naming ?